pkg/storage/container_test.go

```diff
@@ -22,7 +23,7 @@ func Test_Container_Add(t *testing.T) {
 			NewInMemoryCappedContainer(containerCapacity),
 		}
 		for _, c := range containers {
-			t.Run(fmt.Sprintf(" | %s | %s", reflect.TypeOf(c).String(), tt.name), func(t *testing.T) {
+			t.Run(fmt.Sprintf(" | %s | %s", reflect.TypeOf(c).String(), tt.name), func(_ *testing.T) {
```

Comment:
Why `t.Run` when the testing.T on the inside is not used?

Reply:
It's easier to distinguish which test case failed of the whole test run by its name